pkg/convertor.go

```diff
@@ -100,7 +100,9 @@ func computeResourceID(resource parser.TerraformResource) string {
 	case "aws_cloudwatch_log_stream":
 		return fmt.Sprintf("%s:%s", v("log_group_name"), v("name"))
 	case "aws_route":
-		if resource.AttributeValues["destination_cidr_block"] != "" {
+		if resource.AttributeValues["destination_prefix_list_id"] != "" {
```

Comment:
@chiragtayal Given we are returning from the function inside the conditional the "else if" is not required and "if" alone would suffice

i.e. the code would become the following and become slightly more readable

```go
if resource.AttributeValues["destination_prefix_list_id"] != "" { 
	return fmt.Sprintf("%s_%s", v("route_table_id"), v("destination_prefix_list_id"))
} 
if resource.AttributeValues["destination_cidr_block"] != "" {
	return fmt.Sprintf("%s_%s", v("route_table_id"), v("destination_cidr_block"))
}
return fmt.Sprintf("%s_%s", v("route_table_id"), v("destination_ipv6_cidr_block"))
```

Reply:
This change will be done by me post merge